wordpress-mobile/WordPress-Android
# Publish scheduling reminder

This PR connects the FluxC - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1309 and handles the notification shown when the user schedules a post. 

I'll add some more tests in another PR. I wanted to keep this one small.

Expected notification wording is: 
Title: `Scheduled post`
Body: `"[Post name]" has been published`

Title: `Scheduled post: 1 hour reminder`
Body: `"[Post name]" will be published in 1 hour`

Title: `Scheduled post: 10 minute reminder`
Body: `"[Post name]" will be published in 10 minutes` (edited)

To test:
* Go to a Post/Post settings/Publish settings
* Set publish time to the future (current time + 11 minutes)
* Click on "Notification" 
* Select "10 minutes before"
* Wait 2 minutes
* A notification is shown
* Repeat for the other 2 options

To test 2:
* Go to a Post/Post settings/Publish settings
* Set publish time to the future (current time + 11 minutes)
* Click on "Notification"
* Select "1 hour before"
* Click on "Notification"
* Select "10 minutes before"
* Wait 2 minutes
* A notification is shown (the previous notification was cancelled)
* Try with other options as well as setting Notification to "OFF"

Update release notes:
